man/man8/zfs.8

```diff
@@ -3402,6 +3470,38 @@ Give more parsable tab-separated output, without header lines and without arrows
 Display the path's inode change time as the first column of output.
 .RE
 
+.RE
+
+.sp
+.ne 2
+.na
+\fB\fBzfs key -l\fR \fIfilesystem\fR | \fIvolume\fR\fR
+.ad
+.sp .6
+.RS 4n
+Loads a key into ZFS so that the dataset and all of its children that inherit the \fBencryption\fR property may be accessed. The key will be expected in the format and location specified by the \fBkeysource\fR property. Loading a key alone will not automatically mount the dataset. If that functionality is desired, \fBzfs mount\fR will ask for the key and mount the dataset. When a key is loaded the \fBkeystatus\fR property will be set to available.
+.RE
+
+.sp
+.ne 2
+.na
+\fB\fBzfs key -u\fR \fIfilesystem\fR | \fIvolume\fR\fR
+.ad
+.sp .6
+.RS 4n
+Unloads a key from ZFS, removing the ability to access the dataset and all of its children that inherit the \fBencryption\fR property. This requires that the dataset is not currently open or mounted. When a key is unloaded the \fBkeystatus\fR property will be set to \fBunavailable\fR.
+.RE
+
+.sp
+.ne 2
+.na
+\fB\fBzfs key -c\fR [\fB-o\fR \fIkeysource\fR=\fIkeysource\fR] [\fB-o\fR \fIpbkdf2iters\fR=\fIiterations\fR] \fIfilesystem\fR | \fIvolume\fR\fR
+.ad
+.sp .6
+.RS 4n
+Allows a user to change the encryption key used to access a dataset. This command requires that the existing key for the dataset is already loaded into ZFS. This command may also be used to change the \fBpbkdf2iters\fR and / or \fBkeysource\fR properties as needed. If the dataset was previously inheriting the \fBencryption\fR property when this command is run it will now be locally set, indicating that this dataset must have its key loaded separately from the parent.
```

Comment:
See this [comment](https://github.com/zfsonlinux/zfs/pull/4329#discussion_r96513820) I made above.